draft-ietf-anima-autonomic-control-plane/draft-ietf-anima-autonomic-control-plane.xml

```diff
@@ -1198,31 +1198,44 @@ will be distributed with the flooded objective.  An example of the message is in
 
 <t>IPsec tunnel mode is required because the ACP will route/forward packets received from any other ACP node across the ACP secure channels, and not only its own generated ACP packets.  With IPsec transport mode, it would only be possible to send packets originated by the ACP node itself.</t>
 
-<t>IKEv2 authentication MUST use authentication method=1 ("RSA Digital Signature") for ACP certificates with RSA key and methods 9,10,11 for ACP certificates with ECC key according to the keys size/curve.</t>
-
-<t>A certificate payload with the ACP certificate MUST be included during IKEv2 authentication to support the ACP domain membership as described in <xref target="certcheck"/>, because it is using additional elements of the ACP certificates. ACP peers are expected to have the same set of Trust Anchors (TA), so a certificate path MUST only be included in the signaled payload when the path contains intermediate certificates not in the TA set, such as sub-CAs (see <xref target="acp-registrars"/>).</t>
-
-<t>IPsec MUST support ESP with ENCR_AES_GCM_16 (<xref target="RFC4106"/>) due to its higher performance over ENCR_AES_CBC. ACP MUST NOT use any NULL encryption option due to the confidentiality of ACP payload that may not be encrypted by itself (when carrying legacy management protocol traffics as well as hop-by-hop GRASP). These requirements are based on <xref target="RFC8221"/> but limited to the minimum necessary options because there is no need for interoperability with legacy equipment for ACP channels, instead any such backward compatibility would reduce the minimum security or performance required for ACP channels or increase the implementation complexity of the ACP.</t>
+<t>ACP IPsec implementations MUST support ESP with ENCR_AES_GCM_16 (<xref target="RFC4106"/>) due to its higher performance over ENCR_AES_CBC. ACP MUST NOT use any NULL encryption option due to the confidentiality of ACP payload that may not be encrypted by itself (when carrying legacy management protocol traffics as well as hop-by-hop GRASP).  When using AES encryption, 256-bit keys MUST be used. <!--kaduk chacha20/poly1305 is well-liked in other IETF circles but was not previously mentioned for ACP usage.  This text makes no comment about chacha/poly, implicitly allowing it per RFC 8221 but with no specific recommendation for ACP. --> AES-GCM is an AEAD cipher mode, so no ESP authentication algorithm requirement is needed.</t>
 
 <t>Once there are updates to <xref target="RFC8221"/>, these should accordingly be reflected in updates to these ACP requirements, for example if ENCR_AES_GCM_16 was to be superceeded in the future.</t>
 
 <t>Additional requirements from <xref target="RFC8221"/> MAY be used for ACP channels as long as they do not result in a reduction of security over the above MTI requirements. For example, ESP compression MAY be used.</t>
```

Comment:
I'm not sure that "requirements" is the right word, both here and below (for IKEv2).  Mentioning compression makes it sound more like just "IPsec features" that can be used, but it would also be a reasonable interpretation to say that "something listed as MUST- or SHOULD-level in 8221 but not mentioned explicitly here is still allowed in the ACP".  That is, a lot of what we are saying in this section is more of a "mandatory to implement" than a "mandatory to use"; I don't feel properly placed to make a decision about which of those requirements levels is appropriate.